sympy/tensor/array/arrayop.py

```diff
@@ -188,27 +188,49 @@ def derive_by_array(expr, dx):
 
     """
     from sympy.matrices import MatrixBase
+    from sympy.tensor import SparseNDimArray
     array_types = (Iterable, MatrixBase, NDimArray)
-
```

Comment:
why was this line remove?

Reply:
I have seen other examples in NDimArray that there are not new lines between the `import `and the executive code, which made me feel like it is ok to remove it.